packages/web-components/fast-components/src/design-system-provider/index.ts

```diff
@@ -127,7 +127,7 @@ const backgroundStyles = css`
 
 function designToken<T>(token: DesignToken<T>) {
     return (source: DesignSystemProvider, key: string) => {
-        source[key + "Changed"] = function (
+        (source as any)[key + "Changed"] = function (
```

Comment:
This one is probably the one I wish I could fix the most, but really couldn't think of any cleaner way 😢

Reply:
Suppressing this implicit casting error by explicitly casting `as any` seems like a bug. The further I go the more I think setting `"suppressImplicitAnyIndexErrors": true` may be the better option.